### DISCUSSION TITLE
Machine Operator Degraded sync

### DISCUSSION THREAD

**Discussion by okdlukas** [okd-project/okd#2013] [Q&A] [answered]
While still working i have some Machine Config errors on a new bare metal cluster that lack on information how to solve it. 

```
Reprinting Cluster State:
When opening a support case, bugzilla, or issue please include the following summary data along with any other requested information:
ClusterID: 781f6054-0d41-4db8-9505-5f3cb2c2e683
ClientVersion: 4.15.0-0.okd-2024-03-10-010116
ClusterVersion: Stable at "4.15.0-0.okd-2024-03-10-010116"
ClusterOperators:
        clusteroperator/machine-config is degraded because Failed to resync 4.15.0-0.okd-2024-03-10-010116 because: error during syncRequiredMachineConfigPools: [context deadline exceeded, failed to update clusteroperator: [client rate limiter Wait returned an error: context deadline exceeded, error MachineConfigPool worker is not ready, retrying. Status: (pool degraded: true total: 7, ready 7, updated: 7, unavailable: 0)]]

```

**Comment by melledouwsma** [comment, marked as answer]
The quoted output looks like the the result of a `oc adm must-gather` command. Please upload the directory the command produced to a public filesharing service or attach it to this discussion.

If you're unable to upload the results, you can investigate yourself by inspecting `oc describe co machine-config` to find out more about that ClusterOperator. Next step would probably find the degraded MachineConfigPool with `oc get mcp` and then investigate the degraded pool with `oc describe mcp [name-of-degraded-mcp]`.

**Reply by okdlukas** [reply]
thx, 
the error was a machine config that did not go through that was created some time ago. With the right Date and Time i could simply delete it.
